apps/03TicTacToe.js

```diff
@@ -14,6 +14,12 @@ var board = [
 
 var playerTurn = 'X';
 
+function togglePlayerTurn() {
+  playerTurn = (playerTurn === 'X') ? 'X' : 'O';
+}
+
+togglePlayerTurn();
```

Comment:
Debug the flow of this code, is it necessary to toggle the play turn as soon as the game starts